src/androidMain/kotlin/logcat/Throwables.kt

```diff
@@ -11,10 +11,10 @@ import java.io.StringWriter
  * - No silent swallowing of UnknownHostException.
  * - The buffer size is 256 bytes instead of the default 16 bytes.
  */
-fun Throwable.asLog(): String {
-  val stringWriter = StringWriter(256)
-  val printWriter = PrintWriter(stringWriter, false)
-  printStackTrace(printWriter)
-  printWriter.flush()
-  return stringWriter.toString()
+actual fun Throwable.asLog(): String {
```

Comment:
not super familiar with KMP: why is this in androidMain? This should work fine elsewhere, no?

Reply:
huh that's sad.

Reply:
Probably the saddest part of this whole effort - hopefully in the future the JVM + Android can be shared